src/components/section/withImage.stories.tsx

```diff
@@ -3,47 +3,39 @@ import { Meta, StoryObj } from '@storybook/react';
 
 import withoutImageMeta from './withoutImage.stories';
 
-import SectionComponent, { Props } from './index';
+import { Section, SectionProps } from './index';
 
-const Template = ({
-  nativeImageSize = '320x320',
```

Comment:
Could we keep the part where we can see how the bigger image sizes behave in the storybook?